### PR TITLE
Adding names next to fields like "owner" and "publisher"

### PR DESCRIPTION
**Pull request by acron0**
Worried about deploying this though :octopus: 

Please, before :shipit: can some one else test this by loading a master schema, the test data, making some forecasts, then pull this branch and run the migration. Verify using cqlsh or equivalent.

### CODE REVIEW

File: src/witan/app/data.clj [hunk 1]
```diff
@@ -88,19 +92,21 @@
     (first (c/exec (find-data-by-data-id id)))))
 
 (defn data-to-db
-  [{:keys [data-id category name publisher version file-name s3-key public?] :as data}]
+  [{:keys [data-id file-name s3-key public? publisher-name] :as data}]
   (-> data
       (dissoc :data-id
```
Comment (sw1nn) [inline src/witan/app/data.clj]: The `dissoc` is redundant, since the later `assoc` will (effectively) overwrite any values.

Comment (acron0) [inline src/witan/app/data.clj]: You'll notice the key names change from kebab-case to snake_case.

Comment (sw1nn) [inline src/witan/app/data.clj]: Oh, i didn't notice that. fyi, There's `clojure.set/rename-keys` for that, but fine to leave as is.

**Comment by sw1nn** [comment]
My gut feel is that there are some common patterns that will emerge in these migrations but that we don't have enough examples to see them clearly (primary-key-results seems like a thing). Fine to :shipit: but let's revisit the design again when we have more examples.

**Pull request merged**